src/harness/fourslashRunner.ts

```diff
@@ -51,7 +51,7 @@ class FourSlashRunner extends RunnerBase {
         describe(this.testSuiteName + " tests", () => {
             this.tests.forEach((fn: string) => {
                  describe(fn, () => {
-                       fn = ts.normalizeSlashes(fn);
+                        fn = ts.normalizeSlashes(fn);

```

Comment:
This may now be aligned with its' sibling statements, but the indentation in the file is still off, since we're not actually checking it. 🤷‍♀️ It's still better, I suppose.